### ISSUE TITLE
add_adapter

### ISSUE THREAD

**Issue by bleonard** [tpitale/staccato#83]
I'm using 0.5.1 from rubbygems and getting this `undefined method 'add_adapter' for #<Staccato::Tracker:0x00007fa1a3d1a7a8>` from
```ruby
      Staccato.tracker(tracking_id, client_id) do |c|
        c.add_adapter Staccato::Adapter::Logger.new(Staccato.ga_collection_uri, Rails.logger)
        c.add_adapter Staccato::Adapter::Faraday.new(Staccato.ga_collection_uri)
      end
```

I don't see `add_adapter` anywhere in the source of that download, though it is in master.
That's pretty weird. Something happen on publish?

**Comment by tpitale** [comment]
It hasn't been released as a gem version yet. I guess I'll do that now.

**Comment by tpitale** [comment]
Released 0.5.2.

**Comment by bleonard** [comment]
Looking better. Thanks! 🎉
